influxdb/rootfs/Dockerfile

```diff
@@ -1,8 +1,35 @@
-#This dockerfile is based on https://github.com/jalateras/docker-influxdb
-FROM alpine:3.3
-MAINTAINER Jonathan Chauncey "<jchauncey@deis.com>"
+FROM quay.io/deis/base:0.2.0
+
+RUN adduser --system \
+	--shell /bin/bash \
+	--disabled-password \
+	--home /home/influxdb \
+	--group \
+	influxdb
+
+COPY . /
+
+RUN curl -L -o /home/influxdb/envtpl https://github.com/arschles/envtpl/releases/download/0.2.3/envtpl_linux_amd64 \
+	&& chmod +x /home/influxdb/envtpl \
+	&& apt-get update \
+	&& apt-get install -y apt-transport-https \
+	&& curl -sL https://repos.influxdata.com/influxdb.key | apt-key add - \
+	&& . /etc/lsb-release \
+	&& echo "deb https://repos.influxdata.com/${DISTRIB_ID,,} ${DISTRIB_CODENAME} stable" | tee /etc/apt/sources.list.d/influxdb.list \
+	&& apt-get update \
+	&& apt-get install -y influxdb \
```

Comment:
@arschles, @sstarcher and @jchauncey are correct. There isn't really anything pinning it to a specific version of influxdb here. What happens, as you've observed is that the official influxdb package repo gets added to apt-get's sources. We'll end up with whatever is in there.

For example:

```
root@e7909b499db6:/# apt-cache policy influxdb
influxdb:
  Installed: (none)
  Candidate: 0.13.0-1
  Version table:
     0.13.0-1 500
        500 https://repos.influxdata.com/ubuntu xenial/stable amd64 Packages
     0.10.0+dfsg1-1 500
        500 http://archive.ubuntu.com/ubuntu xenial/universe amd64 Packages
```

While this may seem bad, it's no different from how the majority of all our packages are installed-- we get whatever is in the official repo(s).

How badly do we want to fix this?